.github/workflows/python.yml

```
@@ -36,33 +37,22 @@ jobs:
           git config user.email "<>"
           git --version
 
-      - name: Package Python 3.8 and earlier
+      - name: Package Python 3.8 and earlier for Linux x86_64
         if: ${{ startsWith(matrix.os, 'ubuntu') && matrix.pyver == '3.8' }}
         run: docker run --rm -v ${{github.workspace}}/:/io ezralanglois/interop sh /io/tools/package.sh /io /io/dist travis OFF
 
-      - name: Package Python 3.9
-        if: ${{ startsWith(matrix.os, 'ubuntu') && matrix.pyver == '3.9' }}
-        run: docker run --rm -v ${{github.workspace}}/:/io $DOCKER_IMAGE sh /io/tools/package.sh /io /io/dist travis OFF Release cp39-cp39
+      - name: Package Python 3.9-3.x Linux x86_64
+        if: ${{ startsWith(matrix.os, 'ubuntu') }}
+        run: docker run --rm -v ${{github.workspace}}/:/io $DOCKER_IMAGE sh /io/tools/package.sh /io /io/dist travis OFF Release ${{ matrix.pyver }}
 
-      - name: Package Python 3.10
-        if: ${{ startsWith(matrix.os, 'ubuntu') && matrix.pyver == '3.10' }}
-        run: docker run --rm -v ${{github.workspace}}/:/io $DOCKER_IMAGE sh /io/tools/package.sh /io /io/dist travis OFF Release cp310-cp310
-
-      - name: Package Python 3.11
-        if: ${{ startsWith(matrix.os, 'ubuntu') && matrix.pyver == '3.11' }}
-        run: docker run --rm -v ${{github.workspace}}/:/io $DOCKER_IMAGE sh /io/tools/package.sh /io /io/dist travis OFF Release cp311-cp311
-
-      - name: Package Python 3.12
-        if: ${{ startsWith(matrix.os, 'ubuntu') &&  matrix.pyver == '3.12' }}
-        run: docker run --rm -v ${{github.workspace}}/:/io $DOCKER_IMAGE sh /io/tools/package.sh /io /io/dist travis OFF Release cp312-cp312
-
-      - name: Package Python 3.13
-        if: ${{ startsWith(matrix.os, 'ubuntu') &&  matrix.pyver == '3.13' }}
-        run: docker run --rm -v ${{github.workspace}}/:/io $DOCKER_IMAGE sh /io/tools/package.sh /io /io/dist travis OFF Release cp313-cp313
+      - name: Install QEMU
+        if: ${{ startsWith(matrix.os, 'ubuntu') }}
+        run: |
+          docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
 
-      - name: Package Python 3.14
-        if: ${{ startsWith(matrix.os, 'ubuntu') &&  matrix.pyver == '3.14' }}
-        run: docker run --rm -v ${{github.workspace}}/:/io $DOCKER_IMAGE sh /io/tools/package.sh /io /io/dist travis OFF Release cp314-cp314
+      - name: Package Python 3.9-3.x Linux Arm64
```

```suggestion
      - name: Package Python 3.8-3.x Linux Arm64
```
Because of your matrix configuration, this actually also creates a wheel for Python 3.8 (which is good).

---

Ya, will remove the one before. This is redundant